### PR TITLE
Override focus() for text input in IE (to prevent placeholder text from disappearing)

### PR DESCRIPTION
**Pull request by Ujjwalreddy-okta**
OKTA-84944

### CODE REVIEW

File: src/util/BrowserFeatures.js [hunk 1]
```diff
@@ -54,6 +54,10 @@ define(function () {
     return !!(win.history && win.history.pushState);
   };
 
+  fn.isIE = function () {
```
Comment (rchild-okta) [inline src/util/BrowserFeatures.js]: Do we need to worry about edge?

**Comment by rchild-okta** [comment]
Overall looks good - would verify if edge has the same behavior, and if so add some detection for that as well.

**Comment by Ujjwalreddy-okta** [comment]
Edge works as expected. The placeholder text is shown when the field is focused.

**Comment by rchild-okta** [comment]
🚀

**Pull request merged**